projects/netgrif-components/src/lib/data-fields/i18n-field/i18n-text-field/i18n-text-field.component.html

```diff
@@ -4,17 +4,20 @@
             <mat-label *ngIf="!showLargeLayout.value">{{dataField.title}}</mat-label>
             <input matInput
                    [disabled]="true"
-                   [ngModel]="currentValue[selectedLanguage]"
+                   [ngModel]="!!currentValue[selectedLanguage] && currentValue[selectedLanguage] !== ''
```

Comment:
In a logical expression that always results in a boolean, it is not necessary to explicitly convert the value to a boolean.
`!!currentValue[selectedLanguage] && currentValue[selectedLanguage] !== ''`  and  `currentValue[selectedLanguage] && currentValue[selectedLanguage] !== ''` is the same expression.